pages/EventTrail/components/event_trail_map.py

```diff
@@ -0,0 +1,26 @@
+from dash import dcc
+import dash_leaflet as dl
+from . import ids
+
+def render(id):
+    return dcc.Loading(
+        parent_className='loading_wrapper',
+        children=[
+            dl.Map(
+                [dl.TileLayer(), dl.LayerGroup(id=id)],
+                center=[-30, 25], zoom=5,
+                style={'width': '100%', 'height': '500px'}
+            )
+        ],
+        style={'zIndex': 10}
+    )
+
+def generate_markers(data):
+    # Create markers from data frame.

```

Comment:
Does this create the 10 day trail? There is not hovering on these markers that show time yet?